memcached.c

```diff
@@ -7812,6 +7826,7 @@ static int server_sockets(int port, enum network_transport transport,
     }
 }
 
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
not much to be done here :) Maybe when I'm splitting out the code next week I can consolidate some of this into its own file.